src/components/layout/MunicipalityLayout.tsx

```diff
@@ -48,7 +51,7 @@ export function getMunicipalityLayout() {
 function MunicipalityLayout(props: WithChildren) {
   const { children } = props;
   const router = useRouter();
-  const { data } = useSWR(`/json/NL.json`);
+  const { data } = useSWR(`/json/GM0014.json`);
```

Comment:
```suggestion
  const { data } = useSWR<National>(`/json/GM0014.json`);
```
That will statically type the data, saves another cast later on ;)